iso/musl-builder/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV PATH=/home/rust/.cargo/bin:/usr/local/musl/bin:/usr/local/bin:/usr/bin:/bin
 # `--target` to musl so that our users don't need to keep overriding it
 # manually.
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
-    rustup default stable && \
+    rustup default 1.12.1 && \
```

Comment:
Is this version number tied to a specific commit hash? And does the dependency itself have completely fixed dependencies, or is it flexible due to semver?

Reply:
The version number is tied to a specific, non-deterministically built, set of binaries I believe.